src/zmap.c

```diff
@@ -292,8 +292,8 @@ static void start_zmap(void)
 		zconf.probe_module->close(&zconf, &zsend, &zrecv);
 	}
 #ifdef PFRING
-	log_debug("zmap", "destroying zc cluster %p", zconf.pf.cluster);
-	pfring_zc_destroy_cluster(zconf.pf.cluster);
+	//log_debug("zmap", "destroying zc cluster %p", zconf.pf.cluster);
+	//pfring_zc_destroy_cluster(zconf.pf.cluster);
```

Comment:
This code will trigger cpu reset, commented out as a work around, so far doesn't affect our use, the pfring resource actually gets released after the process exit.